.travis.yml

```diff
@@ -41,6 +41,7 @@ jobs:
       node_js: 16
     - stage: eb-deploy-develop
       script:
+        - npm run clean:npm
```

Comment:
When I run this command then zip up files I get a file that 60MB to deploy to AWS.  Is this correct? have you tested this?
```
➜ zip /tmp/agora-app.zip -qr9 * .[^.]*
➜  ll /tmp/agora-app.zip
-rw-r--r--  1 zaro0508  wheel    60M Aug 15 09:42 /tmp/agora-app.zip
```

Reply:
@zaro0508 The small zip generated via the changes in this PR has been manually deployed to the new dev environment, so this should be ready to merge.

Reply:
thanks.  i will test with aws EB CLI.

Reply:
This command fails with `sh: rimraf: command not found`.  Looks like an `npm install` to update dependencies will fix it though.  Could you please do that and commit the resulting `package-lock.json` file with this PR? 

```
➜  npm run clean:npm
> agora@3.0.0 clean:npm
> npm cache clean --force && npm run rimraf -- node_modules

npm WARN using --force Recommended protections disabled.

> agora@3.0.0 rimraf
> rimraf "node_modules"

sh: rimraf: command not found
```

Reply:
@zaro0508 let me know if that's better